packages/manager/apps/catalog-revamp/src/components/Filters/Filters.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useState, useRef } from 'react';
+import {
+  OsdsButton,
+  OsdsText,
+  OsdsIcon,
+  OsdsCheckbox,
+  OsdsCheckboxButton,
+  OsdsLink,
+  OsdsSkeleton,
+} from '@ovhcloud/ods-stencil/components/react/';
+import {
+  OdsIconName,
+  OdsIconSize,
+  OdsButtonVariant,
+  OdsButtonType,
+  OdsTextLevel,
+  OdsButtonSize,
+  OdsCheckboxButtonSize,
+  OdsTextSize,
+} from '@ovhcloud/ods-core';
+import { OdsThemeColorIntent } from '@ovhcloud/ods-theming';
+import { useTranslation } from 'react-i18next';
+import { useCatalog } from '@/hooks/useCatalog';
+
+import './Filters.scss';
+
+const Filters: React.FC = () => {
+  const [selectedCategories, setSelectedCategories] = useState([]);
```

Comment:
you can only create categories, universe and pass it to useCatalog as a props

Reply:
Changed do "categories" and "universes"